torch/ao/quantization/pt2e/utils.py

```diff
@@ -174,14 +174,15 @@ def _is_conv_node(n: Node):
     return n.op == "call_function" and n.target in [
```

Comment:
nit: probably need some renaming here, since `_is_conv_node` is actually `_is_conv_or_conv_transpose_node` now